.github/workflows/rust.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       - uses: actions-rs/clippy-check@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          args: --no-default-features --features "clock" --all-targets # tests need the clock feature
+          args: --no-default-features --lib
```

Comment:
We used to run clippy in `no_std` for the tests, which is not a real requirement. And we weren't even doing that right, since `clock` needs `time/std`, which pulls in `std` anyways.